userComponent/src/main/kotlin/com/gravatar/app/usercomponent/di/UserComponentModule.kt

```diff
@@ -0,0 +1,14 @@
+package com.gravatar.app.usercomponent.di
+
+import com.gravatar.app.usercomponent.data.RealAuthRepository
+import com.gravatar.app.usercomponent.data.WordPressClient
+import com.gravatar.app.usercomponent.domain.repository.AuthRepository
+import org.koin.core.module.dsl.bind
+import org.koin.core.module.dsl.factoryOf
+import org.koin.dsl.module
+
+val userComponentModule = module {
+    factoryOf(::RealAuthRepository) { bind<AuthRepository>() }
+    factoryOf(::WordPressClient)
```

Comment:
I don't see a reason that those should be `singletons`. WDYT?

Reply:
Agree. We are preserving data in those instances, so no need for a singleton. 👌